test/units/plugins/filter/test_ipaddr.py

```diff
@@ -541,3 +539,13 @@ def test_ipsubnet(self):
 
     def _test_ipsubnet(self, ipsubnet_args, expected_result):
         self.assertEqual(ipsubnet(*ipsubnet_args), expected_result)
+
+        expected = 'An Exception indicating a network address is not correctly formatted.'
+        with self.assertRaises(AddrConversionError) as exc:
+            ipsubnet('192.168.144.5', 'invalid_subnet')
+            self.assertEqual(exc.exception.message, expected)
+
+        expected = 'An Exception indicating a failure to convert between address types or notations.'
+        with self.assertRaises(AddrConversionError) as exc:
+            ipsubnet('192.168.144.5', '192.168.8.1.5')
+            self.assertEqual(exc.exception.message, expected)
```

Comment:
This needs to go outside of CM. If `ipsubnet` will raise exception it will interrupt code block within `with`.